test/integration/integration_test.go

```diff
@@ -1364,3 +1369,50 @@ func installDebIfNotExists(r *require.Assertions, container, pkg string) {
 		),
 	))
 }
+
+func testBackupSpecifiedPartition(r *require.Assertions, ch *TestClickHouse) {
+	log.Infof("testBackupSpecifiedPartition started")
+
+	testBackupName := fmt.Sprintf("test_backup_%d", rand.Int())
+	// Create table
+	ch.queryWithNoError(r, "CREATE TABLE default.t1 (dt DateTime, v UInt64) ENGINE=MergeTree() PARTITION BY toYYYYMMDD(dt) ORDER BY dt")
```

Comment:
add DROP IF EXIST here
look like another b2b test contains `t1` table, i will fix it

Reply:
i checked, hm, t1 created only in your test
looks strange